oap-server/server-fetcher-plugin/kafka-fetcher-plugin/src/main/java/org/apache/skywalking/oap/server/analyzer/agent/kafka/provider/handler/LogHandler.java

```diff
@@ -41,25 +41,26 @@
 
     public LogHandler(final ModuleManager moduleManager,
                       final KafkaFetcherConfig config) {
+        super(moduleManager, config);
         this.config = config;
         this.logAnalyzerService = moduleManager.find(LogAnalyzerModule.NAME)
-                                               .provider()
-                                               .getService(ILogAnalyzerService.class);
+                .provider()
```

Comment:
You should import our code style settings file and these changes will not be made

Reply:
ok, this is my negligence. I import the code style and fix it immediately